manifest_staging/charts/gatekeeper/values.yaml

```diff
@@ -6,6 +6,7 @@ createNamespace: true
 disableValidatingWebhook: false
 validatingWebhookTimeoutSeconds: 3
 enableDeleteOperations: false
+experimentalEnableMutation: false
```

Comment:
Since we are going to call this feature alpha (similar to events) in v3.4, do we want to call it `experimental`?

Reply:
From a past GK community call, we landed on `experimentalEnableMutation` as the user-facing field. But I'm happy to change it to something else. NOTE: current alpha features don't have the word `alpha` or `experimental` in them either.

Reply:
Since we are releasing it as alpha, I think we can drop the `experimental` prefix (similar to `events` being alpha).